spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/core/cypher/FilteredPattern.java

```diff
@@ -18,15 +18,32 @@
  */
 package org.neo4j.springframework.data.core.cypher;
 
+import static org.apiguardian.api.API.Status.*;
+
 import org.apiguardian.api.API;
-import org.neo4j.springframework.data.core.cypher.support.Visitable;
+import org.neo4j.springframework.data.core.cypher.support.Visitor;
 
 /**
- * A shared, internally used interface for {@link MapExpression map expressions}.
+ * Used to create patterns filtered via {@literal not}.
  *
  * @author Michael J. Simons
  * @since 1.0
  */
-@API(status = API.Status.INTERNAL, since = "1.0")
-public interface MapEntry extends Visitable {
+@API(status = INTERNAL, since = "1.0")
+final class FilteredPattern implements Condition {
```

Comment:
Why is this called `Filtered...`? Me naming things: `NegatedPattern`

Reply:
See https://neo4j.com/docs/cypher-manual/current/clauses/where/#filter-on-patterns-using-not

Reply:
I think `ExcludedPattern` works best.

> The NOT operator can be used to exclude a pattern.